docs/usage/getting-started/private-packages.md

```diff
@@ -398,6 +399,14 @@ npmRegistries:
     npmAuthToken: <Decrypted PAT Token>
 ```
 
+Thus, to allow for private registries to be used locally via the `NPM_TOKEN` environment variable AND with renovate, your local `.yarnrc.yml` should contain
+```yaml
+npmRegistries:
+   //npm.pkg.github.com/:
+    npmAuthToken: "${NPM_TOKEN}"
+```
+which will be automatically updated by renovate via the hostRules. You *cannot* set a top level `npmAuthToken` in your `.yarnrc.yml` file, as it is not overriden.
```

Comment:
```suggestion

```yaml
npmRegistries:
   //npm.pkg.github.com/:
    npmAuthToken: "${NPM_TOKEN}"
```

The registry will be automatically updated by renovate via the hostRules.
Setting a top level `npmAuthToken` in your `.yarnrc.yml` file is not supported by Renovate, as it is not overriden.
```

Reply:
@rarkins your suggestion is not displaying properly on GitHub.

I think GitHub does not like it when you have backticks within backticks, like in a review suggestion. I can't get those kind of review suggestion to work properly either. :upside_down_face: